src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -120,26 +124,6 @@ import FileViewField, {
   App107959cDocReviewField,
 } from '../components/File/FileViewField';
 
-// Used to condense some repetitive schema boilerplate
-const maxApplicants = 3;
-const applicantListSchema = (requireds, propertyList) => {
-  return {
-    type: 'object',
-    properties: {
-      applicants: {
-        type: 'array',
-        minItems: 1,
-        maxItems: maxApplicants,
-        items: {
-          type: 'object',
-          required: requireds,
-          properties: propertyList,
-        },
-      },
-    },
-  };
-};
-
```

Comment:
Relocated to helper/utility file because we need it in the newly imported marriage date schema.